src/main/kotlin/app/revanced/patches/shared/settings/preference/impl/ListPreference.kt

```diff
@@ -14,16 +14,16 @@ import org.w3c.dom.Element
  * @param title The title of the list preference.
  * @param entries The human-readable entries of the list preference.
  * @param entryValues The entry values of the list preference.
- * @param default The default entry value of the list preference.
  * @param summary The summary of the list preference.
+ * @param default The default entry value of the list preference.
  */
 internal class ListPreference(
     key: String,
     title: StringResource,
     val entries: ArrayResource,
     val entryValues: ArrayResource,
-    val default: String? = null,
-    val summary: StringResource? = null
+    val summary: StringResource? = null,
+    val default: String? = null, // Default does not need to be set if key is present in YouTube or TikTok SettingsEnum
```

Comment:
I think the default should not be nullable just because Twitch does not utilize it, instead it should map the structure of the XML preferences. SwitchPreference doesn't necessarily need "default" to be present in its args, thus can be nullable.